src/tools/ToolPanZoom.ts

```diff
@@ -0,0 +1,10 @@
+import BaseTool, { ToolConstructor } from './BaseTool';
+
+/**
+ * Enables pan and zoom functionality in svgPanZoom.
+ */
+const ToolPanZoom: ToolConstructor = class ToolPanZoom extends BaseTool {
+
```

Comment:
Let's make an issue to add the hand cursor when this tool is selected, and when mouse down, the cursor should be grab.

Reply:
https://github.com/calband/calchart-redesign/issues/65